sled-agent/src/instance.rs

```diff
@@ -239,6 +239,9 @@ enum InstanceRequest {
         ip: InstanceExternalIpBody,
         tx: oneshot::Sender<Result<(), ManagerError>>,
     },
+    RefreshExternalIps {
+        tx: oneshot::Sender<Result<(), ManagerError>>,
+    },
```

Comment:
I think in future, we can cut a lot of this circuitous movement through `InstanceRunner` out (including EIP attach/detach) if we place external IP state onto NetworkInterface directly.

Definitely out of scope here given the immediacy of this PR!